tests/sentry/api/endpoints/test_source_map_debug_blue_thunder_edition.py

```diff
@@ -0,0 +1,1405 @@
+                    exceptions=[create_exception_with_frame({"abs_path": "/some/path/to/file.js"})],
+                    debug_meta_images=[
+                        {
+                            "type": "sourcemap",
+                            "code_file": "/some/path/to/file.js",
+                            "debug_id": "8d65dbd3-bb6c-5632-9049-7751111284ed",
+                        }
+                    ],
+                ),
+                project_id=self.project.id,
+            )
+            resp = self.get_success_response(
+                self.organization.slug,
+                self.project.slug,
+                event.event_id,
+            )
+            assert resp.data["has_debug_ids"]
+
+    def test_has_debug_ids_false(self):
+        with self.feature("organizations:source-maps-debugger-blue-thunder-edition"):
+            event = self.store_event(
+                data=create_event(
+                    exceptions=[create_exception_with_frame({"abs_path": "/some/path/to/file.js"})],
+                    debug_meta_images=None,
+                ),
+                project_id=self.project.id,
+            )
+            resp = self.get_success_response(
+                self.organization.slug,
+                self.project.slug,
+                event.event_id,
+            )
+            assert not resp.data["has_debug_ids"]
+
+    def test_sdk_version(self):
+        with self.feature("organizations:source-maps-debugger-blue-thunder-edition"):
+            event = self.store_event(
+                data=create_event(sdk={"name": "sentry.javascript.react", "version": "7.66.0"}),
+                project_id=self.project.id,
+            )
+            resp = self.get_success_response(
+                self.organization.slug,
+                self.project.slug,
+                event.event_id,
+            )
+            assert resp.data["sdk_version"] == "7.66.0"
+
+    def test_no_sdk_version(self):
+        with self.feature("organizations:source-maps-debugger-blue-thunder-edition"):
+            event = self.store_event(data=create_event(), project_id=self.project.id)
+            resp = self.get_success_response(
+                self.organization.slug,
+                self.project.slug,
+                event.event_id,
+            )
+            assert resp.data["sdk_version"] is None
+
+    def test_sdk_debug_id_support_full(self):
+        with self.feature("organizations:source-maps-debugger-blue-thunder-edition"):
+            event = self.store_event(
+                data=create_event(sdk={"name": "sentry.javascript.react", "version": "7.66.0"}),
+                project_id=self.project.id,
+            )
+            resp = self.get_success_response(
+                self.organization.slug,
+                self.project.slug,
+                event.event_id,
+            )
+            assert resp.data["sdk_debug_id_support"] == "full"
+
+    def test_sdk_debug_id_support_needs_upgrade(self):
```

Comment:
l (not specifically related to this PR but I noticed it while reviewing): Does this mean we should update our docs? Currently they all call out 7.47.0 as the minimum version. The wizard also checks for 7.47.0.

Reply:
Yes! I have this on my todo list!